README.md

```diff
@@ -1,46 +1,40 @@
-# Getting Started with Create React App
+# This is Jack's HomeTask for 10/23
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## This is a 2-player game of Connect Four with TypeScript and React..
 
-## Available Scripts
+This will help you spending you mean time enjoyable. Having `Fun`!
 
-In the project directory, you can run:
+## Techstacks
 
-### `npm start`
+`React`, `TailwindCss`, `Babel`, `Typescript`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Requirements
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+1. **The board should be 7 wide by 6 high**
+2. **It should alternate between the 'red' and 'yellow' players turn**
+3. **Clicking the "Drop" button should drop a token of the current player's to the bottom-most free position**
+4. **Clicking the "Drop" button on a full column should have no effect**
+5. **If either player gets four in a row (horizontally, vertically, or diagonally)**
+   1. The "COLOR's turn" heading should be replaced by "COLOR won!"
+   2. The "Drop" buttons should be replaced by a "Play again" button
+6. **If the board is full and there is no winner**
+   1. The "COLOR's turn" heading should be replaced by "Draw!"
+   2. The "Drop" buttons should be replaced by a "Play again" button
 
-### `npm test`
+## Most Challenging Things
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+1. **Replace the 'Drop' button with a representation of the current player's token, e.g. if it's red's turn, show a semi-transparent red token above each column.**
+   So, I checked if there are any tokens in columns and showed those semi-transparent tokens.
+2. **Keep track of and display how many times each player has won. Save it so the score is tracked even when reloading the page.**
+   I kept that data inside local storage and solved this problem.
+3. **Try to implement the `checkForWinner` function yourself: it should check for four tokens in a row in the horizontal, vertical, and diagonal directions.**
+   I created a checker function that checks if that game status matches the finishing condition.
```

Comment:
It would better to pick up just one challenge and dig into the brief solution about it.